main/python/rootls.py

```diff
@@ -41,25 +42,29 @@
 
 - rootls -t example.root
   Display contents of the ROOT file 'example.root', use a long listing format and print trees recursively.
+
+- rootls -r example.root
+  Display contents of the ROOT file 'example.root', traversing recursively any TDirectory.
 """
 
 def get_argparse():
-	parser = cmdLineUtils.getParserFile(description, EPILOG)
-	parser.prog = 'rootls'
+    parser = cmdLineUtils.getParserFile(description, EPILOG)
+    parser.prog = 'rootls'
 
-	parser.add_argument("-1", "--oneColumn", help=ONE_HELP, action= "store_true")
-	parser.add_argument("-l", "--longListing", help=LONG_PRINT_HELP, action= "store_true")
-	parser.add_argument("-t", "--treeListing", help=TREE_PRINT_HELP, action= "store_true")
-	return parser
+    parser.add_argument("-1", "--oneColumn", help=ONE_HELP, action= "store_true")
+    parser.add_argument("-l", "--longListing", help=LONG_PRINT_HELP, action= "store_true")
+    parser.add_argument("-t", "--treeListing", help=TREE_PRINT_HELP, action= "store_true")
+    parser.add_argument("-r", "--recursiveListing", help=RECURSIVE_PRINT_HELP, action= "store_true")
+    return parser
 
 
 def execute():
-	parser = get_argparse()
-	# Put arguments in shape
-	sourceList, optDict = cmdLineUtils.getSourceListOptDict(parser)
+    parser = get_argparse()
+    # Put arguments in shape
+    sourceList, optDict = cmdLineUtils.getSourceListOptDict(parser)
 
-	# Process rootLs
-	return cmdLineUtils.rootLs(sourceList, oneColumn=optDict["oneColumn"], \
-							longListing=optDict["longListing"], treeListing=optDict["treeListing"])
+    # Process rootLs
+    return cmdLineUtils.rootLs(sourceList, oneColumn=optDict["oneColumn"], \
+                            longListing=optDict["longListing"], treeListing=optDict["treeListing"], recursiveListing=optDict["recursiveListing"])
```

Comment:
```suggestion
    return cmdLineUtils.rootLs(sourceList, oneColumn=optDict["oneColumn"],
                            longListing=optDict["longListing"], treeListing=optDict["treeListing"], recursiveListing=optDict["recursiveListing"])
```
I think you don't need the `\` here (in fact it would also code-formatting tools if we eventually want to run them on this file).